README.adoc

```diff
@@ -1028,8 +1028,8 @@ end
 
 === Context Descriptions
 
-When an example block description is composed with `context` block descriptions, it should result in a sentence with proper grammar.
-This typically means the `context` should start with a term such as 'when', 'with', or 'without'.
+When an example block description is composed with `context` block descriptions, it should form a sentence with proper grammar.
+This typically means the `context` should be an adjunct adpositional phrase, such as one starting with 'when', 'with', 'without', 'before', 'after', or similar.
```

Comment:
As a non-linguist, I had to look "adjunct adpositional phrase" in the Internet. Do you think it's a good wording for this?